packages/expo-contacts/ios/Decoding.swift

```diff
@@ -69,7 +68,7 @@ func decodeUrlAddresses(_ input: [UrlAddress]?) -> [CNLabeledValue<NSString>]? {
   var output = [CNLabeledValue<NSString>]()
   for item in input {
     let label = decodeUrlAddressLabel(item.label)
-    if let urlString = item.url?.path {
+    if let urlString = item.url?.formatted() {
```

Comment:
CI is failing because `formatted` is only available on iOS 16 and above. Can you add an availability check?

Reply:
I'll make it soon

Reply:
Finally I use `absoluteString`